index.js

```diff
@@ -27,7 +27,7 @@ function createError (code, message, statusCode = 500, Base = Error) {
       this.cause = args.pop().cause
     }
 
-    this.message = format(message, ...args)
+    this.message = format(message, ...args).trim()
```

Comment:
Not sure which one will be better, most of the time the `if` clause will be executed.

```suggestion
    if (message)  args.unshift(message)
    this.message = format(...args)
```

Reply:
I would:
1. `if (typeof message !== 'string') throw Error('Fastify error message must be defined')` (line 11)
2. `if (message === '') { this.message = message } else { this.message = format(message, ...args) }`